MAINTAINING.md

```diff
@@ -34,17 +34,19 @@ the overall validity of the PR.
     Is the problem that the PR intends to solve clearly stated?
 2.  Is the solution presented in the PR as simple as possible to satisfy the
     requirements, but no simpler?
-3.  Is the solution well designed on a high level?
-4.  Do the concepts used by the PR make sense?
-5.  Does the PR break with existing concepts?
-6.  Is the structure of the PR itself valid?
-7.  Are there clear and adequate instructions on how to test the PR? \
+3.  Is the PR functionally atomic? Does it address a single issue or should
```

Comment:
Now that it is two questions: mind splitting it up like in the rest of the list?

Reply:
Some of the other points are phrased as two related questions - IMO it's more helpful to group them...

Reply:
What do you think? (It's good that these are getting well discussed.)

Reply:
I can put a line break there, to have the structure match point 1 and 8.